client/disputes/index.tsx

```diff
@@ -99,55 +103,72 @@ const headers = [
 	},
 ];
 
-export const DisputesList = () => {
+export const DisputesList = (): JSX.Element => {
+	/* eslint-disable */
 	const { disputes, isLoading } = useDisputes( getQuery() );
 
+	const {
+		disputesSummary,
+		isLoading: isSummaryLoading,
+	} = useDisputesSummary();
+
 	const rows = disputes.map( ( dispute ) => {
```

Comment:
I think this is probably better:

```suggestion
	const rows = disputes.map( ( {
			dispute_id,
			amount = 0,
			currency,
			reason,
			source,
			order_number,
			customer_name,
			customer_email,
			customer_country,
			status,
			created,
			due_by,
		} ) => {
```